target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -146,6 +146,12 @@ case "$FIRMWARE" in
 			/lib/firmware/ath10k/QCA9888/hw2.0/board.bin
 		rm /lib/firmware/ath10k/QCA9888/hw2.0/board-2.bin
 		;;
+	comfast,cf-ew72)
+		caldata_extract "art" 0x5000 0x2f20
+		ln -sf /lib/firmware/ath10k/pre-cal-pci-0000\:00\:00.0.bin \
+		/lib/firmware/ath10k/QCA9888/hw2.0/board.bin
```

Comment:
Please use proper indent here. Does the interface get a proper MAC address?

Reply:
Will do.  MAC addresses are 20:0D:B0:75:9B:F4 for the 2.4G and 20:0D:B0:75:9B:F5 for the 5G.

Reply:
Okay, so the one strange thing is that the MAC addresses are backwards (stock F4 is for the 5G, for this it looks like F5 is for the 5G though and F4 is for the 2.4G).  Any advice to fix?